http-serverless/samples/sample1/src/main/resources/META-INF/native-image/serialization-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "types":[
+  ],
+  "lambdaCapturingTypes":[
+  ],
+  "proxies":[
+  ]
+}
```

Comment:
why is this native mage configuration needed?

Reply:
removed this and other such empty configs